docs/general/builtins/window-functions.rst

```diff
@@ -106,13 +106,17 @@ In ``ROWS`` mode ``CURRENT_ROW`` means the current row.
 The ``offset PRECEDING`` and ``offset FOLLOWING`` options vary in meaning
 depending on the frame mode. In ``ROWS`` mode, the ``offset`` is an integer
 indicating that the frame start or end is offsetted by that many rows before or
-after the current row. In ``RANGE`` mode, the use of a custome ``offset``
+after the current row. In ``RANGE`` mode, the use of a custom ``offset``
 option requires that there is exactly one ``ORDER BY`` column in the window
 definition. The frame contains those rows whose ordering column value is no
 more than ``offset`` minus (for PRECEDING) or plus (for FOLLOWING) the current
 row's ordering column value. In this case the data type of the ``offset``
 expression must have the same type of the ordering column.
 
+.. note::
+    When the ordering column is of type :ref:`timestamp <timestamp_data_type>`,
```

Comment:
Maybe it would make more sense to re-phrase the last sentence of the previous paragaph. To explain that the offset value is substracted ot added to the ordering value, and so only type combinations which support add/substract are supported.